tests/queries/0_stateless/01744_fuse_sum_count_aggregate.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS fuse_tbl;
+CREATE TABLE fuse_tbl(a Int8, b Int8) Engine = Log;
+INSERT INTO fuse_tbl SELECT number, number + 1 FROM numbers(1, 20);
+
+set optimize_fuse_sum_count_avg = 1;
```

Comment:
By the way, is it always faster? Maybe we can enable it by default, so that it is better tested?

Reply:
>This setting should not be enabled by default to avoid incompatibilities in clusters where new and old server versions may work at the same time.

Ah, OK.